CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 * Added `databricks_repo` resource to manage [Databricks Repos](https://docs.databricks.com/repos.html) ([#771](https://github.com/databrickslabs/terraform-provider-databricks/pull/771))
 
+**Deprecations**
+* `azure_workspace_name`, `azure_resource_group`, `azure_subscription_id`, and `azure_workspace_resource_id` are deprecated and would be removed in v0.4.0. Please rewrite provider configuration with `host = data.azurerm_databricks_workspace.example.workspace_url` to achieve the same effect. Please check [azurerm_databricks_workspace](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/databricks_workspace#workspace_url) resource documentation for details.
```

Comment:
should we change it to `host = azurerm_databricks_workspace.example.workspace_url (or host = data.azurerm_databricks_workspace.example.workspace_url)` ?

the first variant is primarily used when chaining azurerm with our provider